paper.md

```diff
@@ -17,93 +17,98 @@ authors:
   - name: Joe McGlinchy
     orcid: 0000-0003-2135-0168
     affiliation: 1
-  - name: Jenny Palomino 
+  - name: Jenny Palomino
     orcid: 0000-0003-4879-9299
     affiliation: 1
   - name: Korinek, Nathan
     orcid: 0000-0003-0859-7246
     affiliation: 1
-  - name: Chris Holdgraf 
+  - name: Chris Holdgraf
     orcid: 0000-0002-2391-0678
     affiliation: 2
+  - name: Tim Head
+    orcid: 0000-0003-0931-3698
+    affiliation: 3
 affiliations:
   - name: "Earth Lab, University of Colorado - Boulder"
     index: 1
   - name: "University of California - Berkeley, Project Jupyter"
     index: 2
+  - name: "Tim's Affiliation Here"
```

Comment:
@betatim can you please suggest what you'd like your affiliation(s) to be here?? i can then merge this and earthpy will be in JOSS!!